src/components/Map.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Map as LeafletMap, WMSTileLayer } from 'react-leaflet';
+
+/**
+ * Leaflet map
+ *
+ * @param height - map container height
+ * @param maps - defines different tile layers
+ * @param position - initial position [lat, long]
+ * @param zoom - initial zoom [2..21]
+ * @returns map element
+ *
+ * Example:
+ * <Map
+ *   position={[60.4629060928519,22.259694757206415]}
+ *   zoom={12}
+ *   height={400}
+ *   maps={{
+ *     wms: {
+ *       layers: ['Opaskartta'],
+ *       url: 'https://opaskartta.turku.fi/TeklaOGCWeb/WMS.ashx',
+ *     }
+ *   }}
+ * />
+ *
+ */
+
+const Map = ({height, maps, position, zoom}) => {
+  return (
+    <LeafletMap center={position} zoom={zoom} style={{height: height}}>
+      {maps.wms && <WMSTileLayer
+        layers={maps.wms.layers}
```

Comment:
Hm. I was thinkng that we will supporte many different types of provider. If we know that `wms` will be the only type, than we can rewrite the code like
```
+const Map = ({height, layers, position, url, zoom}) => {
+  return (
+    <LeafletMap center={position} zoom={zoom} style={{height: height}}>
+      <WMSTileLayer
+        layers={layers}
+        url={url}
+      />}
+    </LeafletMap>
+  );
+};
```